### ISSUE TITLE
Janitorial keyring item bugging after an interrupted interaction

### ISSUE THREAD

**Issue by El-furro** [ParadiseSS13/Paradise#21433]
**Issue Description**: 
The janitorial keyring item bugs after a failed interaction with a door (in my case, the door was pried open with the jaws of life before i could finish my interacting). It will start stating that you are already using the item on whatever door you interact with regardless of if you were prior.

**What did you expect to happen**: 
I expect it to tell me that i am unlocking the door with the keyring when initially using it on a door that it has access too

**What happened instead**: 
It will tell me that i am already using it on said door regardless of if i was prior

**Why is this bad/What are the consequences**: 
The item becomes useless as it will not let you unlock any doors

**Steps to reproduce the problem**: 
I used the keys on a door i had access too when i saw somebody attempting to pry said door open with the jaws of life, after which the item started working as intended (i clicked on the door multiple times, leading to the fail message playing correctly when it should have), but the action was interrupted as the door was opened before the keys could open it, leading to the bugged state.
(i'm not sure if it was the jaws opening the door, or just the door opening in general that causes the problem)

**When did the problem start happening**:

**Extra information**:
